vendor/BUILD

```diff
@@ -8154,6 +8154,7 @@ go_test(
     srcs = [
         "k8s.io/apimachinery/pkg/api/resource/amount_test.go",
         "k8s.io/apimachinery/pkg/api/resource/math_test.go",
+        "k8s.io/apimachinery/pkg/api/resource/quantity_proto_test.go",
```

Comment:
@k82cn I think the issue was that when I ran update-bazel.sh it updated 50+ docs and I didn't know where to look to find the actual update.